payment_paypal/indico_payment_paypal/controllers.py

```diff
@@ -84,14 +89,22 @@ def _verify_business(self):
         return False
 
     def _verify_amount(self):
-        expected_amount = self.registration.price
+        paypal_fixed_fee = float(current_plugin.event_settings.get(self.registration.registration_form.event,
+            'paypal_fixed_fee'))
+        paypal_percent_fee = float(current_plugin.event_settings.get(self.registration.registration_form.event,
+            'paypal_percent_fee'))
+        expected_amount = float(self.registration.price)
```

Comment:
avoid using floats for anything payment-related, since these can introduce tiny rounding errors in larger numbers.. please always use the `Decimal` type

Reply:
done